relay/writer/file.go

```diff
@@ -135,7 +135,7 @@ func (w *FileWriter) Flush() error {
 	if w.out != nil {
 		return w.out.Flush()
 	}
-	return nil
+	return errors.Errorf("no underlying writer opened")
```

Comment:
Just use `errors.New()` if you aren't formatting anything.

Reply:
my fault, addressed.